package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bancor/contracts-solidity",
-    "version": "0.6.32",
+    "version": "1.22.10",
```

Comment:
this is the contracts version, no? shouldn't it stay as 0.6.32 (or 0.6.33)?

Reply:
Oh, my bad. I'll revert